notebooks/official/tensorboard/tensorboard_custom_training_with_custom_container.ipynb

```diff
@@ -29,7 +29,7 @@
         "id": "ed2pOXQMb8fY"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    ! gsutil -m rm -r {BUCKET_URI}
</pre><p> </p><p>use template version for deleting bucket</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/818/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>